src/Nav/Nav.js

```diff
@@ -1,10 +1,10 @@
 import './Nav.css';
-import { NavLink } from 'react-router-dom'
+import { NavLink, Link } from 'react-router-dom'
 
 const Nav = () => {
   return (
     <nav>
-      <h1 id='nav-title'>humor me</h1>
+      <Link to='/'><h1 id='nav-title'>humor me</h1></Link>
```

Comment:
Always love adding this detail by making the 'logo' clickable to bring the user to home